src/prometheus_alert_rules/patroni_rules.yaml

```diff
@@ -1,12 +1,17 @@
+# This file based on https://samber.github.io/awesome-prometheus-alerts/rules#patroni-1
+
 groups:
-  - name: PatroniExporterK8s
-
-    rules:
-      - alert: PatroniPostgresqlDown
-        expr: "patroni_postgres_running == 0"
-        for: 0m
-        labels:
-          severity: critical
-        annotations:
-          summary: Patroni Posrgresql Down (instance {{ $labels.instance }})
-          description: "Patroni Postgresql instance is down\n  VALUE = {{ $value }}\n  LABELS = {{ $labels }}"  
```

Comment:
This is no longer available in a new rules files (and it was also missing in VM version of rules).

Q: Should `PatroniPostgresqlDown` still exist? IMHO, yes. Otherwise we do not alert if Partoni is down.

Can you please restore this rule for both VM and K8s alerts.

Reply:
Hi! I restored this rule